grammars/blade.cson

```diff
@@ -11,21 +11,25 @@
     'patterns': [
       {
         'begin': '{{--'
-        'captures':
+        'beginCaptures':
           '0':
-            'name': 'punctuation.definition.comment.blade'
+            'name': 'punctuation.definition.comment.begin.blade'
         'end': '--}}'
+        'endCaptures':
+          '0':
+            'name': 'punctuation.definition.comment.end.blade'
         'name': 'comment.block.meta.embedded.blade'
       }
       {
         'begin': '(?<!@){{{'
-        'captures':
+        'beginCaptures':
           '0':
-            'name': 'support.punctuation.blade'
-          '1':
-            'name': 'source.php'
```

Comment:
As a way to trigger PHP-specific snippets and commands, I assume?

Why not move the `source.php` scope to the containing block's `name` property, then...?